SingularityService/src/main/java/com/hubspot/singularity/data/transcoders/JsonTranscoder.java

```diff
@@ -14,7 +14,7 @@
   private final ObjectMapper objectMapper;
   private final Class<T> clazz;
 
-  JsonTranscoder(final ObjectMapper objectMapper, final Class<T> clazz) {
+  public JsonTranscoder(final ObjectMapper objectMapper, final Class<T> clazz) {
```

Comment:
wasn't sure if this was OK -- i assume it is, but wasn't sure if it was package private on purpose...